automation/publish.sh

```diff
@@ -11,6 +11,7 @@
 source automation/check-patch.setup.sh
 cd ${TMP_PROJECT_PATH}
 
+export PLATFORMS=all
```

Comment:
this is bit tricky, because it will run on post submit, so harder to test it
we can just hope for best and do manual tests before that / fix according needs